users/users_router.js

```diff
@@ -0,0 +1,20 @@
+const express = require("express")
+
+const userDB = require("../users/users-db")
+// const restricted = require("../util/authorization")
+
+const router = express.Router()
+
+router.get("/", async (req, res) => {
+  try {
+    users = await userDB.getAllUsers()
+    if (users.length > 0) res.status(200).json(users)
+    else res.status(401).json({ message: "No users found" })
+  } catch {
```

Comment:
You shouldn't use status 401 here. You're just trying to retrieve users, not check if the user is properly authenticated